.github/workflows/test.yml

```diff
@@ -18,6 +18,6 @@ jobs:
       run: make patch
     - name: Generate
       run: |
-        make clean gen mod fmt patch-post docs move-other test
+        make generate
         # Expect all changes to be accounted for
-        ! git status --porcelain | grep .
+        ! git status --porcelain | grep -v -e 'go.mod' -e 'go.sum'
```

Comment:
Nit:  I'm a bit concerned about ignoring uncommitted changes to `go.mod` and `go.sum`; I'll merge this as-is for now but we should keep an eye out for issues due to missed updates for these files (or revert this change in a follow-up PR).

Reply:
I understand that completely. Maybe I remove that change with the overall Fabric SDK release update. 

The issue was that two packages had releases that were causing GHA failures even though there wasn't any issue with it. My thought was that they would need to be excluded from the GHA diff check because some changes to the repo wouldn't be a full SDK update and the GHA will always just check if the output differs between full builds.